.ci/cico_rhche_prcheck.sh

```diff
@@ -65,18 +66,6 @@ fi
 if [ -z "${RH_CHE_AUTOMATION_RDU2C_USERNAME}" ] || [ -z "${RH_CHE_AUTOMATION_RDU2C_PASSWORD}" ]; then
   echo "RDU2C credentials not set"
   CREDS_NOT_SET="true"
```

Comment:
It's cool that you have moved all precondition checks higher in the script and that you exit before continuing with `oc login`. However why don't you `exit 2` here (as well as at lines 64 and 72) instead of continuing execution? Code would be shorter, you could get rid of a global variable (`CREDS_NOT_SET`) and we would fail even faster.

Reply:
Because I want to see which credentials are incorrect. every check has it's own `echo`. This thing has been already discussed in the PR I've opened against Ilja's repo.

Reply:
So why don't you leave the specific `echo` for each error but than `exit 2` right away?

Reply:
I'm confused
I am doing the checks in 3 separate if statements
and as soon as they are done with the check they do `exit 2`